### ISSUE TITLE
Add rule for appsealing

### ISSUE THREAD

**Issue by kiror0** [rednaga/APKiD#133]
https://www.appsealing.com/

I haven't looked up to it yet, might be something new.

**Comment by enovella** [comment]
Hi @zeroload,

do you have a sample to dig into?

Cheers

**Comment by zeroload** [comment]
@enovella Nope, but this link should provide some sample apps https://www.appsealing.com/references-en/

**Comment by zeroload** [comment]
Some interesting files I've found so far.
```
libcovault-appsec.so
assets/appsealing.dex
assets/sealed1.dex
```

I'll do PR at some point.

**Comment by enovella** [comment]
Could you do it before 5-6th December? Planning to release a new APKiD version.

**Comment by zeroload** [comment]
Uh. Sorry for the delay, I managed to get some info from this packer, but unfortunately I dropped my laptop last week (right after some progress with breakdown the packer). It'll be a while before it gets fixed. Thus, I only can submit my not-so-good progress on this packer from phone.

**Comment by apkunpacker** [comment]
request to open this issue again , for me appsealing not detected

**Comment by strazzere** [comment]
@apkunpacker the noted one in this issue? Or other ones? If other ones - can you please attach them or a list of hashes?

**Comment by apkunpacker** [comment]
i have other samples which stay undetected . sure i will upload sample. should i open new issue or continue in this ?

**Comment by strazzere** [comment]
Here is fine - if need be, for variant reasons, I can reorganize elsewhere.

**Comment by enovella** [comment]
> i have other samples which stay undetected . sure i will upload sample. should i open new issue or continue in this ?

can you verify that's detected now?  @apkunpacker 

Feel free to re-open if not

**Comment by apkunpacker** [comment]
yeah . it detected already sometime ago . it start detecting after you merged pr #223 :+1:
